include/lsst/afw/math/Random.h

```diff
@@ -126,23 +125,6 @@ class Random final {
      *      Thrown if memory allocation for internal generator state fails.
      */
     explicit Random(std::string const &algorithm, unsigned long seed = 1);
-    /**
-     * Creates a random number generator using the algorithm and seed specified
-     * in the given policy. The algorithm name and seed are expected to be specified
-     * in string-valued keys named "rngAlgorithm" and "rngSeed" respectively. The
-     * "rngSeed" value is expected to be convertible to an unsigned long integer
-     * and must not be positive.
-     *
-     * @param[in] policy    policy which contains the algorithm and seed to
-     *                      to use for random number generation
-     * @returns              a newly created random number generator
-     *
-     * @throws lsst::pex::exceptions::InvalidParameterError
-     *      Thrown if the requested algorithm is not supported.
-     * @throws std::bad_alloc
-     *      Thrown if memory allocation for internal generator state fails.
-     */
-    explicit Random(std::shared_ptr<pex::policy::Policy> const policy);
```

Comment:
Is this at L54-55 no longer valid?
```
... (or the "rngAlgorithm" and "rngSeed" keys in a policy)
```

Reply:
Also a good catch - I think that whole paragraph has probably been invalid for a while, because those factory functions do not seem to exist.